spec/factories/user.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    name { 'User 1' }
```

Comment:
Use [faker](https://github.com/stympy/faker) gem to create fake data for testing purposes